system/modules/core/library/Contao/Database.php

```diff
@@ -594,15 +594,15 @@ public function getUuid()
 
 
 	/**
-	 * Quote the column name if it a reserved word
+	 * Quote an identifier if it is a reserved word
 	 *
 	 * @param string $strName
 	 *
 	 * @return string
 	 */
-	public static function quoteColumnName($strName)
+	public static function quoteIdentifier($strName)
 	{
-		if (in_array(strtolower($strName), array('rows'), true))
+		if ($strName == 'rows')
```

Comment:
@leofeyer why did you remove the `strtolower()`? 
This would make it no longer work for `Rows` or `ROWS`.